test/mocks/mockWebContext.js

```diff
@@ -36,6 +36,7 @@ const mockWebContext = (userLang = 'en') => {
           ladokUID: 'd',
           language: 'English',
           roundId: '2',
```

Comment:
`roundId` has to be removed in all tests

Reply:
Done